package.json

```diff
@@ -45,6 +46,7 @@
 		"typescript": "^3.0.3"
 	},
 	"dependencies": {
+		"strong-tools": "^5.2.0",
```

Comment:
Should only be required as a devDependency right?

Reply:
Yes, sorry, didn't notice that when npm said `--dev` was "deprecated" it meant "ignored".